src/components/AddEditProfile/AddEditProfile.jsx

```diff
@@ -0,0 +1,92 @@
+import './AddEditProfile.css';
+import { useUser } from '../../context/UserContext';
+import { createProfile, updateProfile } from '../../services/profiles';
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useProfile } from '../../hooks/useProfile';
+
+export default function AddEditProfile({ isCreating = false }) {
+  const [name, setName] = useState('');
+  const [birthday, setBirthday] = useState('');
+  const [bio, setBio] = useState('');
+  const { user } = useUser();
+  const { profile, setProfile } = useProfile();
+  const history = useHistory();
+
+  useEffect(() => {
+    setName(profile.name);
+    setBirthday(profile.birthday);
+    setBio(profile.bio);
+  }, [profile]);
```

Comment:
A useEffect may not be necessary here, we could just set the default state to these values on page load